get_lims_info.py

```diff
@@ -1,15 +0,0 @@
-#!/usr/bin/env python
-
-from genologics import lims
-from config import URI, USER, PASSWORD
-
-def get_reference_id(sample_name):
-	"""
-	Get reference genome for a sample from LIMS
-	Inputs: Sample ID
-	Outputs: Reference genome - NCBI accession number
-	"""
-	
-	l = lims.Lims(URI, USER, PASSWORD)
```

Comment:
it's more readable to use variable names that are meaningful
I try to avoid single character variable names :)